### ISSUE TITLE
html.elements.ins - Safari with VoiceOver does not announce text preceding `<ins>` element

### ISSUE THREAD

**Issue by aardrian** [mdn/browser-compat-data#27445]
### What type of issue is this?

Incorrect support data (example: BrowserX says "86" but support was added in "40")

### What information was incorrect, unhelpful, or incomplete?

Support for the `<ins>` element is incomplete / has regressed in Safari 18 (through 18.5 as of this writing). When paired with VoiceOver it functionally removes preceding content from the page.

### What did you expect to see?

When using VoiceOver with Safari, that the text preceding a `<ins>` would be announced.'

### Did you test this? If so, how?

In a test using `<ins>` or `role="insertion"`, VoiceOver with Safari does not announce the opening of the sentence, instead jumping to where the element starts.

To replicate:

1. Open Safari and activate VoiceOver
2. Go to:
   * https://cdpn.io/aardrian/debug/rNRxZmP
   * https://adrianroselli.com/demos/text-level-semantics-reprised/
3. Use the `read-all` command (VO + A)
4. Observe how, for most paragraphs, the words preceding the element are not announced.

### Can you link to any release notes, bugs, pull requests, or MDN pages related to this?

[286267: AX: VoiceOver does not announce text preceding some phrasing elements](https://bugs.webkit.org/show_bug.cgi?id=286267)

### Do you have anything more you want to share?

https://github.com/user-attachments/assets/c7a8a68d-903a-45d5-b3e8-fe6553c0c7c9

### MDN URL

https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/ins

### MDN metadata

<details>
<summary>MDN page report details</summary>

* Query: `html.elements.ins`
* Report started: 2025-07-29T21:49:44.502Z

</details>

**Comment by aardrian** [comment]
Related:
* https://github.com/mdn/browser-compat-data/issues/27443
* https://github.com/mdn/browser-compat-data/issues/27444
* https://github.com/mdn/browser-compat-data/issues/27445
* https://github.com/mdn/browser-compat-data/issues/27446

**Comment by caugner** [comment]
Closing as won't fix, see https://github.com/mdn/browser-compat-data/issues/27443#issuecomment-3179926464.
